src/test/java/org/junitpioneer/platform/testkit/engine/PioneerEngineExecutionResults.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.platform.testkit.engine;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.platform.engine.TestExecutionResult;
+import org.junit.platform.engine.discovery.DiscoverySelectors;
+import org.junit.platform.testkit.engine.EngineExecutionResults;
+import org.junit.platform.testkit.engine.EngineTestKit;
+import org.junit.platform.testkit.engine.Events;
+
+/**
+ * Pioneers own class to handle {@link org.junit.platform.testkit.engine.EngineExecutionResults} of the JUnit-Jupiter-Engine.
+ *
+ */
+public class PioneerEngineExecutionResults {
+
+	EngineExecutionResults executionResults;
```

Comment:
Is this package-private on purpose?

Reply:
Yes I always use package private instead of private to acces fields and methods from tests

Reply:
But in this case - do you access it? I'm not seeing the tests for this class.